src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/resources/ExperimentResource.java

```diff
@@ -368,7 +387,8 @@ private Experiment updateExperimentGapStop(int id, Experiment experiment, Experi
         newPopulations.forEach(population -> {
             try {
                 platformManager.publishTask(population.getPlatformId(), old).join();
-                storePopulation(id, population);
+                //TODO marcel: update?
```

Comment:
yes the behaviour here is that we will only add the population to the list of populations if the publishing was successfull.